views/extras.tmpl

```diff
@@ -547,6 +565,17 @@
     });
 
     $("#keynav-toggle .text").text(keyNavEnabled() ? "Disable Hot Keys":"Enable Hot Keys" )
+
+    var scrollPosition
+
+    Turbolinks.reload = function () {
+        scrollPosition = [window.scrollX, window.scrollY]
+        Turbolinks.visit(window.location, { action: 'replace' })
+        if(!ws){
+            createWebSocket(window.location)
```

Comment:
please add a console log inside this if statement so we know if this is happening.